docs/metrics/customizing-the-sdk/README.md

```diff
@@ -403,6 +403,61 @@ AnotherFruitCounter.Add(4, new("name", "mango"), new("color", "yellow")); // Not
 streams. There is no ability to apply different limits for each instrument at
 this moment.
 
+### Exemplars
+
+Exemplars are example data points for aggregated data. They provide access to
+the raw measurement value, time stamp when measurement was made, and trace
+context, if any. It also provides "Filtered Tags", which are those attributes
+(Tags) [dropped by a view](#select-specific-tags). Exemplars are an opt-in
+feature, and allows customization via ExemplarFilter and ExemplarReservoir.
+
+#### ExemplarFilter
+
+`ExemplarFilter` determines which measurements are eligible to become an
+Exemplar. i.e. `ExemplarFilter` determines which measurements are offered to
+`ExemplarReservoir`, which makes the final decision about whether the offered
+measurement gets stored as an exemplar. They can be used to control the noise
+and overhead associated with Exemplar collection.
+
+OpenTelemetry SDK comes with the following Filters:
+
+* `AlwaysOnExemplarFilter` - makes all measurements eligible for being an Exemplar.
+* `AlwaysOffExemplarFilter` - makes no measurements eligible for being an
+  Exemplar. Use this to turn-off Exemplar feature.
+* `TraceBasedExemplarFilter` - makes those measurements eligible for being an
+Exemplar, which are recorded in the context of a sampled parent `Activity`
+(span).
+
+`SetExemplarFilter` method on `MeterProviderBuilder` can be used to set the
+desired `ExemplarFilter`.
+
+The snippet below shows how to set `ExemplarFilter`.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    // rest of config not shown
+    .SetExemplarFilter(new TraceBasedExemplarFilter())
+    .Build();
+```
+
+> **Note**
+> As of today, there is no separate toggle for enable/disable Exemplar
+feature. It can be turned off by using `AlwaysOffExemplarFilter`.
```

Comment:
This note makes it sound like Exemplars would be turned on by default and user has to explicitly turn it off by using `AlwaysOffExemplarFilter`. Was that the intended message?

Reply:
agree it require some refinement.
The end goal is:
1. There will be a separate On/Off for Exemplars feature as a whole, with OFF by default.
2. The default Filter will be TraceBasedExemplarFilter.

Not there yet as I am using Filter as a way to on/off things now :)